src/applications/Machine.java

```diff
@@ -0,0 +1,53 @@
+package applications;
+
+import dataStructures.LinkedQueue;
+
+class Machine {
+    // data members
+    private LinkedQueue jobQ; // queue of waiting jobs for this machine
+    private int changeTime; // machine change-over time
+    private int totalWait; // total delay at this machine
+    private int numTasks; // number of tasks processed on this machine
+    private Job activeJob; // job currently active on this machine
+
+    // constructor
+    Machine() {
+        jobQ = new LinkedQueue();
+    }
+
+    public LinkedQueue getJobQ() {
+        return jobQ;
+    }
+
+    public int getChangeTime() {
+        return changeTime;
+    }
+
+    public void setChangeTime(int changeTime) {
+        this.changeTime = changeTime;
+    }
+
+    public int getTotalWait() {
+        return totalWait;
+    }
+
+    public void setTotalWait(int totalWait) {
```

Comment:
We should see if we can eliminate this setter, or at least make it so that it's purpose (why we would want to overwrite the `totalWait`) is clearer. I'm guessing that we want to increment this by adding a value that is passed in as an argument, but I'm not 100% sure of that.